### PR TITLE
feat: upgraded flatpak to org.freedesktop.Platform version 23.08

### PR DESCRIPTION
**Pull request by themr0c**
references https://github.com/containers/podman-desktop/issues/3966

### CODE REVIEW

**Comment by benoitf** [comment]
https://github.com/flathub/org.electronjs.Electron2.BaseApp/issues/43

seems you shouldn't update to 23.08 if base is not 23.08 as well

**Comment by themr0c** [comment]
So first thing, a 23.08 branch in https://github.com/flathub/org.electronjs.Electron2.BaseApp/branches?

**Comment by themr0c** [comment]
See: https://github.com/flathub/org.electronjs.Electron2.BaseApp/issues/43

File: io.podman_desktop.PodmanDesktop.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 app-id: io.podman_desktop.PodmanDesktop
 runtime: org.freedesktop.Platform
-runtime-version: "22.08"
+runtime-version: "23.08"
 sdk: org.freedesktop.Sdk
 base: org.electronjs.Electron2.BaseApp
 base-version: "22.08"
```
Comment (themr0c) [inline io.podman_desktop.PodmanDesktop.yml:6]:
```suggestion
base-version: "23.08"
```

**Comment by themr0c** [comment]
Okay then this will be able to build once baseapp 23.08 is out.

File: io.podman_desktop.PodmanDesktop.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 app-id: io.podman_desktop.PodmanDesktop
 runtime: org.freedesktop.Platform
-runtime-version: "22.08"
+runtime-version: "23.08"
 sdk: org.freedesktop.Sdk
 base: org.electronjs.Electron2.BaseApp
-base-version: "22.08"
+base-version: "23.08"
```
Comment (themr0c) [inline io.podman_desktop.PodmanDesktop.yml:6]:
```suggestion
base-version: "23.08"
```

**Comment by themr0c** [comment]
Rebased to trigger a rebuild.

**Comment by bbhtt** [comment]
bot, build

**Comment by bbhtt** [comment]
bot, build

**Comment by benoitf** [comment]
bot, build

**Pull request merged**